ces_container/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Use the official Python 3.10 image as the base image  
+FROM python:3.10
+
+# Set the working directory to /app
+WORKDIR /app
+
+# Copy the requrements file
+COPY requirements.txt /app/requirements.txt
```

Comment:
Do we need to have the same set of dependencies? Maybe only a limited subset for kernel is enough?